spec/relativetimeformat.html

```diff
@@ -310,37 +328,29 @@ <h1>PartitionRelativeTimePattern ( _relativeTimeFormat_, _value_, _unit_ )</h1>
           1. If _patterns_ has a field [[&lt;_valueString_&gt;]], then
             1. Let _result_ be _patterns_.[[&lt;_valueString_&gt;]].
             1. Return a List containing the Record { [[Type]]: *"literal"*, [[Value]]: _result_ }.
-        1. If _value_ is *-0*<sub>𝔽</sub> or if _value_ is less than 0, then
+        1. If _value_ is *-0*<sub>𝔽</sub> or _value_ &lt; *-0*<sub>𝔽</sub>, then
           1. Let _tl_ be *"past"*.
         1. Else,
           1. Let _tl_ be *"future"*.
         1. Let _po_ be _patterns_.[[&lt;_tl_&gt;]].
-        1. Let _fv_ be ! PartitionNumberPattern(_relativeTimeFormat_.[[NumberFormat]], _value_).
-        1. Let _pr_ be ! ResolvePlural(_relativeTimeFormat_.[[PluralRules]], _value_).[[PluralCategory]].
+        1. Let _fv_ be PartitionNumberPattern(_relativeTimeFormat_.[[NumberFormat]], ℝ(_value_)).
+        1. Let _pr_ be ResolvePlural(_relativeTimeFormat_.[[PluralRules]], _value_).[[PluralCategory]].
         1. Let _pattern_ be _po_.[[&lt;_pr_&gt;]].
-        1. Return ! MakePartsList(_pattern_, _unit_, _fv_).
+        1. Return MakePartsList(_pattern_, _unit_, _fv_).
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-makepartslist" aoid="MakePartsList">
-      <h1>MakePartsList ( _pattern_, _unit_, _parts_ )</h1>
+    <emu-clause id="sec-makepartslist" type="abstract operation">
+      <h1>
+        MakePartsList (
+          _pattern_: a pattern String,
+          _unit_: a String,
+          _parts_: a List of Records representing a formatted Number,
+        ): a List of Records with fields [[Type]] (a String), [[Value]] (a String), and [[Unit]] (a String or ~empty~)
```

Comment:
I think we've mostly tried to introduce named types when Records get to be at all nontrivial. E.g. you can have a `<dfn variants="TimePart Records">TimePart Record</dfn>` or whatever, giving names/types/descriptions for these fields, and then this AO would return `A List of TimePart Records` and it would link to that section.